src/plugins/banner/public/index.scss

```diff
@@ -43,4 +43,21 @@ $expandedHeaderWidth: $headerWidth * 2;
   top: calc(var(--global-banner-height) + $headerWidth);
 }
 
+.headerGlobalNav--withBanner {
+  #globalHeaderBars {
+    position: relative;
+    top: var(--global-banner-height);
+  }
+}
+
+.headerGlobalNav--withBanner .newTopNavHeader {
+  z-index: 900; // lower than banner
```

Comment:
What banner are we talking about here? Can we define the banner z-index as constant, and use something similar to `$ouiZHeader - 1` to avoid hardcoded z-index number?